src/Voter/UriPrefixVoter.php

```diff
@@ -51,6 +76,11 @@ public function matchItem(ItemInterface $item)
 
         $content = $item->getExtra('content');
 
+        $request = $this->request;
+        if (null !== $this->requestStack) {
+            $request = $this->requestStack->getMasterRequest();
+        }
+/* no idea what to do here ... & beyond */        
         if ($content instanceof Route && $content->hasOption('currentUriPrefix')) {
             $currentUriPrefix = $content->getOption('currentUriPrefix');
             $currentUriPrefix = str_replace('{_locale}', $this->request->getLocale(), $currentUriPrefix);
```

Comment:
You should replace each ocurence of `$this->request` (like here) by your request.